index.html

```diff
@@ -155,9 +177,7 @@ <h2 class="recentwork">
             <li><a href="#">Javascript</a></li>
             <li><a href="#">html</a></li>
           </ul>
-          <button class="resumebutton buttons" type="submit">
-            See Project
-          </button>
+          <button class="resumebutton buttons" type="submit">See Project</button>
```

Comment:
- [ ] Kindly add some space on top of the button the improve its visibility and to match the design in Figma 👍

## Your design:
![image](https://user-images.githubusercontent.com/50721479/206475427-53d785eb-eb85-45f8-aaeb-4a71a17aa5ef.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/206475463-a78dd129-cca5-4148-a171-ce7adac1db5b.png)